src/redux/FilmSlice/FilmType.tsx

```diff
@@ -16,5 +16,9 @@ export interface Film {
 export interface FilmList {
   list: Film[];
   isLoading: boolean;
-  value: number;
+  page: number;
+  total: number;
+  nextPageUrl: null | string;
+  prevPageUrl: null | string;
```

Comment:
Please explain me the logic behind this ?

Reply:
I forgot to remove, I removed this issue